tests/python/gaia-ui-tests/gaiatest/apps/base.py

```diff
@@ -19,8 +19,11 @@ def __init__(self, marionette):
         self.apps = GaiaApps(self.marionette)
         self.accessibility = Accessibility(self.marionette)
         self.frame = None
-        self.manifest_url = hasattr(self, 'manifest_url') and self.manifest_url or None
```

Comment:
That's the only occurrence of this variable in the whole patch. I don't see a reason of keeping it :)